onthequad/onthequad/UserEndPoints.py

```diff
@@ -0,0 +1,89 @@
+import falcon
+import json
+from .UsersDB import UsersDB
+from .DataBaseSetUp import DataBaseSetUp
+authen = DataBaseSetUp.authentication()
+
+class UserEndPoints:
+
+    def on_get(self, req, resp):
+        
+        send = UsersDB.getAllUsers()
+        resp.body = json.dumps(send)
+        resp.status = falcon.HTTP_200
+
+    def on_post(self, req, resp):
+        data = json.loads(req.stream.read())
+    
+        if any(data):
+
+            firstName = data['firstName']
```

Comment:
https://www.tutorialspoint.com/python/python_dictionary.htm
A tutorial on dictionaries. There is no need to take all the values out and put them back in again.